src/ios-deploy/ios-deploy.m

```diff
@@ -72,11 +72,13 @@
 const char* error_path = NULL;
 
 typedef struct am_device * AMDeviceRef;
+typedef void (*AMDeviceInstallCallback)(NSDictionary<NSString *, id> *callbackDictionary, void *device);
```

Comment:
Fairly sure this should be a CFDictionary.

Reply:
I picked this definition up the way it is and it likely worked due to auto bridging from NS to CF but since the rest of the code here uses CF, I changed this definition to match the one above and verified it is still calling the proper mobiledevice function.

Reply:
I'm still seeing
`typedef void (*AMDeviceInstallCallback)(NSDictionary<NSString *, id> *callbackDictionary, void *device);`
did something not get pushed up?

Reply:
Missed this one, sorry about that. It no longer needs its own callback definition so removed this typedef.